Code/Framework/AzFramework/AzFramework/Viewport/ViewportScreen.cpp

```diff
@@ -91,16 +88,17 @@ namespace AzFramework
     {
         const auto worldFromView = AzFramework::CameraTransform(cameraState);
         const auto cameraWorldTransform = AZ::Transform::CreateFromMatrix3x3AndTranslation(
-            AZ::Matrix3x3::CreateFromMatrix4x4(worldFromView), worldFromView.GetTranslation());
+            AZ::Matrix3x3::CreateFromMatrix3x4(worldFromView), worldFromView.GetTranslation());
         return AZ::ViewFrustumAttributes(
             cameraWorldTransform, AspectRatio(cameraState.m_viewportSize), cameraState.m_fovOrZoom, cameraState.m_nearClip,
             cameraState.m_farClip);
     }
 
-    AZ::Vector3 WorldToScreenNdc(const AZ::Vector3& worldPosition, const AZ::Matrix4x4& cameraView, const AZ::Matrix4x4& cameraProjection)
+    AZ::Vector3 WorldToScreenNdc(const AZ::Vector3& worldPosition, const AZ::Matrix3x4& cameraView, const AZ::Matrix4x4& cameraProjection)
     {
         // transform the world space position to clip space
-        const auto clipSpacePosition = cameraProjection * cameraView * AZ::Vector3ToVector4(worldPosition, 1.0f);
+        const auto clipSpacePosition =
+            cameraProjection * AZ::Matrix4x4::CreateFromMatrix3x4(cameraView) * AZ::Vector3ToVector4(worldPosition, 1.0f);
```

Comment:
This is not required, you can leave the old code, it's actually cheaper than building new matrix and doing the same multiplication.